sdk/keyvault/azure-security-keyvault-jca/src/test/resources/aad/invalid-access-token-response.json

```diff
@@ -0,0 +1,9 @@
+{
+  "token_type": "Bearer",
+  "expires_in": "3599",
```

Comment:
So the key difference is `expires_in` or `ext_expires_in`?

Reply:
I was wrong, please ignore this file.